squad/frontend/templates/squad/build.html

```diff
@@ -53,6 +55,7 @@
 {% for test_run in build.test_runs.all %}
 {% for status in test_run.status.all %}
 {% if status.suite and status.has_metrics %}
+<a href="{% project_url test_run %}#metrics-{{status.suite}}">
```

Comment:
This might be a risky thing to do in terms of HTML validity
https://stackoverflow.com/questions/1827965/is-putting-a-div-inside-an-anchor-ever-correct

Reply:
I just checked againt the [w3c validator](https://validator.w3.org/) and it passed. I just needed to added a few missing markup items to make it 100% valid HTML5 (thus commit  80f48d2)